nodes/quantum-algorithms/portfolio-optimisation/portfolio-optimisation.html

```diff
@@ -0,0 +1,78 @@
+<script type="text/javascript">
+  RED.nodes.registerType("portfolio-optimisation", {
+    category: "quantum algorithms",
+    color: "#A8A8A8", 
+    defaults: {
+      name: { value: "" },
+      solverMethod: { 
+        value: "", 
+        required: true
+      },
+      seeds:{
+        value: 123,
+        required: true
+      },
+      assets:{
+        value: 4,
+        required: true
+      },
+      start:{
+        value: 2020-01-01,
+        required: true
+      },
+      end:{
+        value: 2021-01-01,
+        required: true
+      }
+    },
+    inputs: 1,
+    outputs: 1,
+    icon: "font-awesome/fa-microchip",
+    align: "left", 
+    paletteLabel: "portfolio optimisation node",
+    label: function () {
+      return this.name || "portfolio optimisation";
+    },
+    inputLabels: "Connect Injector here",
+    outputLabels: "Connect Debug here",
```

Comment:
The node doesn't necessarily have to use the inject and debug nodes. I think it would be better to just put "input" and "output" here if there isn't any specific way of describing it. Also, for consistency keep the text fully lowercase.